swift/apple/FirezoneKit/Sources/FirezoneKit/AuthClient/AuthClient.swift

```diff
@@ -95,6 +121,34 @@ private final class WebAuthenticationSession: NSObject,
     }
   }
 
+  static func createRandomHexString(byteCount: Int) throws -> String {
```

Comment:
This would probably be a good candidate to unit test.
```suggestion
  // TODO: Unit tests
  static func createRandomHexString(byteCount: Int) throws -> String {
```

Reply:
@jamilbk What should we test for in a unit test here? That the byte count is correct?

Reply:
Yeah, these should suffice:
- test that two created strings differ
- test the length

That way if someone changes this down the line they can be sure it won't break what's relying on it